### PR TITLE
Removing 'target_type' and 'target_value' parameters - KNTK-291

### PR DESCRIPTION
**Pull request by ziemowit141**
Removing outdated parameters as per discussion in: https://github.com/kentik/terraform-provider-kentik-synthetics/pull/23#discussion_r670674544

Ticket: https://clkentik.atlassian.net/browse/KNTK-291

### CODE REVIEW

**Comment by ziemowit141** [comment]
`test-data.json ` contained those parameters so I removed them. I think stub server changed a lot in test-data.json. Wasn't sure wheather I should commit those changes, or not. That is why the diff is so huge.

File: synthetics/test-data.json [hunk 1]
```diff
@@ -95,14 +95,10 @@
           "server-one",
           "server-two"
         ],
-        "targetType": "dummy-tt",
-        "targetValue": "dummy-tv",
-        "useLocalIp": false,
```
Comment (Fenthick) [inline synthetics/test-data.json:100]: I'd prefer you not to modify lines other than those with targetType and targetValue.
This file is based on real payloads returned by the server, so modifying them would make it less realistic. I have no strong opinion about that, tough.

**Review by Fenthick — CHANGES_REQUESTED** [review]
Also, please add ticket number to the PR description

**Comment by mmac-m3a** [comment]
`golangci-lint` is failing with this PR: https://github.com/kentik/terraform-provider-kentik-synthetics/actions/runs/1037526906

File: synthetics/test-data.json (hunk 1 above)
Comment (ziemowit141) [reply synthetics/test-data.json:100]: After `make test` this fille is changed automatically in many places. I should ignore those changes and commit its state before i run `make test`?

Comment (Fenthick) [reply synthetics/test-data.json:100]: Yes

Comment (ziemowit141) [reply synthetics/test-data.json:100]: Addressed with latest commit

**Review by Fenthick — APPROVED** [review]

**Review by mmac-m3a — APPROVED** [review]

**Review by mmac-m3a — APPROVED** [review]

**Pull request merged**